MyMovies/View Controllers/MovieSearchCell.swift

```diff
@@ -0,0 +1,50 @@
+//
+//  MovieSearchCell.swift
+//  MyMovies
+//
+//  Created by Chad Parker on 5/1/20.
+//  Copyright © 2020 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class MovieSearchCell: UITableViewCell {
+    
+    var movieRepresentation: MovieRepresentation? {
+        didSet {
+            updateViews()
+        }
+    }
+
+    @IBOutlet weak var movieTitleLabel: UILabel!
+    @IBOutlet weak var saveButton: UIButton!
+    
+    override func awakeFromNib() {
+        super.awakeFromNib()
+    }
+
+    override func setSelected(_ selected: Bool, animated: Bool) {
+        super.setSelected(selected, animated: animated)
+        // Configure the view for the selected state
+    }
```

Comment:
Before pushing your final code make sure to get rid of functions you are not using.